check-submodules.sh

```diff
@@ -44,10 +44,11 @@ echo "$SUBMODULE_NAME-old: $OLD"
 echo "$SUBMODULE_NAME-new: $NEW"
 
 # If metrics directories differ in number of files,
-# print the difference of files
+# print only the files that are in a directory but not in the other one
 if [ $OLD != $NEW ]
 then
-    diff -rq /tmp/$SUBMODULE_NAME-old /tmp/$SUBMODULE_NAME-new
+    ONLY_FILES=`diff -q /tmp/$SUBMODULE_NAME-old /tmp/$SUBMODULE_NAME-new | grep "Only in"`
```

Comment:
Don't we still need the `-r` option?

Reply:
We can remove the `-r` option since the recursion are not necessary for the directories we are scanning through. They are flat